frontend/src/components/FormRadioBinaryChoice.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Form } from "react-bootstrap";
+
+export default ({
+  children,
+  isDefaultChoice = false,
+  name,
+  onlyChoice = false,
+  onSelect,
+  selected,
+  title
+}) => (
+  <div
+    className={`radio-card ${selected && "selected"}`}
+    style={{
+      width: "49%",
```

Comment:
nts what this is doing but it looks like something that could be better achieved with flexbox? difficult to say without knowing what the problem was but I'm always wary when I see things like 49%!

Reply:
So you can see an example of this component here:
![Screenshot 2020-12-17 at 02 34 21](https://user-images.githubusercontent.com/1789893/102436413-68fd0980-4010-11eb-9bc0-b4daa711257e.png)

To refactor given I have a similar binary radio choice later (all Data mappers vs specify Data Mappers), I created this component here `FormRadioBinaryChoice`. As you can see in the following line, I have 49% of width + 1% of margin right (that is what I already had in the previous iteration).

The only new thing here is adding this param `onlyChoice` where in case I add 51% because `Composite` is not an available option and I basically don't want this to happen:
![Screenshot 2020-12-17 at 02 41 20](https://user-images.githubusercontent.com/1789893/102436867-6222c680-4011-11eb-973f-7f7ee3b57978.png)

I'm not super familiar with flexbox but I can try to dig in if you think it could be a good improvement. 
What do you think?

Reply:
ah ok I see what you mean. probably. a good candidate for something like bootstraps responsive grid system. I was worrying that when the screen is small it'll look a bit weird at 49%?